app/src/main/java/com/example/Egida/Dependencies.kt

```diff
@@ -1,14 +1,21 @@
 package com.example.Egida
 
-import com.example.Egida.data.Database
-import com.example.Egida.domain.useCase.UserUSeCaseImpl
-import com.example.Egida.domain.useCase.UserUseCase
+import com.example.Egida.data.DatabaseAUTH
+import com.example.Egida.data.DatabaseUser
+import com.example.Egida.domain.useCase.UserDBUseCase
+import com.example.Egida.domain.useCase.userAUTH.UserAUTHUSeCaseImpl
+import com.example.Egida.domain.useCase.userAUTH.UserAUTHUseCase
+import com.example.Egida.domain.useCase.userDB.UserDBUseCaseImpl
 
 
 object Dependencies {
 
-    private val database: Database by lazy { Database() }
+    private val DATABASE_AUTH: DatabaseAUTH by lazy { DatabaseAUTH() }
+    private val DATABASE_USER: DatabaseUser by lazy { DatabaseUser() }
 
-    fun getUserUseCase(): UserUseCase =
-        UserUSeCaseImpl(database)
+    fun userAUTHUseCase(): UserAUTHUseCase =
```

Comment:
No need to write AUTH in upper case. Even for abbriviation common comunity used rule: follow camel case.